snow/engine/avalanche/vertex/heap.go

```diff
@@ -3,133 +3,27 @@
 
 package vertex
 
-import (
-	"container/heap"
-
-	"github.com/ava-labs/avalanchego/ids"
-	"github.com/ava-labs/avalanchego/snow/consensus/avalanche"
-	"github.com/ava-labs/avalanchego/utils/set"
-)
-
-var (
-	_ Heap           = (*maxHeightVertexHeap)(nil)
-	_ heap.Interface = (*priorityQueue)(nil)
-)
-
-type priorityQueue []avalanche.Vertex
-
-func (pq priorityQueue) Len() int {
-	return len(pq)
-}
-
-// Returns true if the vertex at index i has greater height than the vertex at
-// index j.
-func (pq priorityQueue) Less(i, j int) bool {
-	statusI := pq[i].Status()
-	statusJ := pq[j].Status()
+import "github.com/ava-labs/avalanchego/snow/consensus/avalanche"
```

Comment:
consider moving the Less function to the file where we define vertex and dropping this file.